portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,124 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width:1000px;
+  color:#777;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  background-color: lightblue;
+  margin-left:auto;
+  margin-right:auto;
 }
 
-#greeting-container {
+#pix {
+    text-align:center;
+    background-color: none;
+}
+
+h1{
```

Comment:
Similarly, let's add spaces between the CSS selector and the bracket (e.g. `h1 {` instead of `h1{`).